TAF/utils/scripts/docker/common-taf.env

```diff
@@ -1,5 +1,6 @@
 # Common variables
 DOCKER_HOST_IP=172.17.0.1
 
-# Scalability test export data to northbound MQTT broker with MQTT_BROKER_IP
+# Scalability test export data to northbound MQTT broker
 MQTT_BROKER_IP=172.17.0.1
```

Comment:
are we still using this? if not, please remove it.

Reply:
Script for Scalability test is still using it. The variable can be removed after the related script updated.